core/chains/evm/txmgr/finalizer.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/rpc"
 
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils"
```

Comment:
Think this would be better to group with the other `evmtypes` core/chains import below

Reply:
I was getting lint [error](https://github.com/smartcontractkit/chainlink/actions/runs/10275365121/job/28468383610), so I ran the `goimports` command on this file specifically. It's strange the IDE should automatically format it..

Reply:
Huh did it have a space between the 2 imports? goimports is keeping them grouped for me when I try.